docs/01-app/04-api-reference/01-directives/use-cache.mdx

```diff
@@ -140,11 +190,9 @@ export default function Page() {
 
 > This is recommended for applications that previously used the [`export const dynamic = "force-static"`](/docs/app/api-reference/file-conventions/route-segment-config#dynamic) option, and will ensure the entire route is prerendered.
```

Comment:
Do we want to mention about dynamic usage further down the tree impacting this?

Reply:
Good point, I think it's more important than the line on `force-static`. This feels more like a migration detail rather than part of the main body.